invenio/templates/deployments/worker.yaml

```diff
@@ -102,16 +109,27 @@ spec:
         volumeMounts:
         - mountPath: /opt/invenio/var/instance/data
           name: shared-volume
+        {{- range .Values.worker.volumeMounts_extra }}
+        - name: {{ .name }}
+          mountPath: {{ .mountPath }}
+          subPath: {{ default "" .subPath }}
+        {{- end }}
         {{- end }}
         {{- if .Values.worker.resources }}
         resources: {{- toYaml .Values.worker.resources | nindent 10 }}
         {{- end }}
-      {{- if .Values.persistence.enabled }}
       volumes:
+      {{- if .Values.persistence.enabled }}
```

Comment:
That changes the intention of the `if`-statement: if `.Values.persistence.enables` is false and there are no extra volumes defined this leads to an empty `volumes` tag. Might be problematic but never tried that.